src/collective/cover/static/css/cover.css

```diff
@@ -249,30 +249,23 @@ table.invisible
 }
 
 /* Tile Calendar */
-.cover-calendar-tile .calendar-tile-header
+.cover-calendar-tile .portletCalendar
```

Comment:
@hvelarde I don't understand your concern here with this css code.. this selector is restrict to our calendar tile because of the ``.cover-calendar-tile`` class.  It will not  affect the portlet calendar.